levels.go

```diff
@@ -1040,19 +1040,35 @@ func (s *levelsController) getTableInfo(withKeysCount bool) (result []TableInfo)
 		l.RLock()
 		for _, t := range l.tables {
 			var count uint64
+			var smallest, biggest []byte
+
+			// Create copies of smallest and biggest.
+			smallest = append(smallest, t.Smallest()...)
+			biggest = append(biggest, t.Biggest()...)
```

Comment:
There could be a table with only `badgerPrefix` keys. What should we do in that case? Skip the table?